resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -127,6 +121,11 @@ private[spark] class Client(
             .pods()
             .withLabels(driverKubernetesSelectors)
             .watch(podWatcher)) { _ =>
+          val probePingHttpGet = new HTTPGetActionBuilder()
+            .withScheme(if (driverSubmitSslOptions.enabled) "HTTPS" else "HTTP")
```

Comment:
It does need to be capitalized, the Kubernetes API will throw an error otherwise.